nacc/uds3/filters.py

```diff
@@ -63,16 +92,59 @@ def filter_replace_drug_id(input_ptr, filter_meta, output_ptr):
 def filter_fix_c1s(input_ptr, filter_meta, output_ptr):
 
     lines = input_ptr.read().splitlines()
+    output = csv.writer(output_ptr, dialect=csv.excel)
     header = True
     for line in lines:
         if header:
             header = False
             for key in fix_c1s_headers.keys():
+                print >> sys.stderr, 'key : ' + key + ' Value : '+  fix_c1s_headers[key]
                 line=line.replace(key, fix_c1s_headers[key],1)
-        print line
+        output_ptr.write(line)
+        output_ptr.write("\n")
     return
 
+def filter_fix_fvpheader(input_ptr, filter_meta, output_ptr):
+
+    lines = input_ptr.read().splitlines()
+    header = True
+    for line in lines:
+        if header:
+            header = False
+            for key in fix_fvp_headers.keys():
+                print >> sys.stderr, 'key : ' + key + ' Value : '+  fix_fvp_headers[key]
+                line=line.replace(key, fix_fvp_headers[key],1)
+        output_ptr.write(line)
+        output_ptr.write("\n")
+
+    return
+
+
+def filter_remove_ptid(input_ptr, filter_meta, output_ptr):
+
+    reader = csv.DictReader(input_ptr)
+    output = csv.DictWriter(output_ptr, None)
+    write_headers(reader, output)
+    for record in reader:
+        prog = re.compile("11\d.*")
```

Comment:
This needs to be configurable. Since this tool is used outside of UF, other organizations may not have the same 11* numbering convention we have.